numba/dataflow.py

```diff
@@ -181,7 +181,8 @@ def op_BUILD_MAP(self, info, inst):
 
     def op_BUILD_SET(self, info, inst):
         count = inst.arg
-        items = [info.pop() for _ in range(count)]
+        # Note: related python bug http://bugs.python.org/issue26020
+        items = list(reversed([info.pop() for _ in range(count)]))
```

Comment:
Can also be written as `[info.pop() for _ in range(count)][::-1]` (just saying :-))